### PR TITLE
Add `source` property to `position`

### PR DESCRIPTION
**Pull request by lydell**
This allows for source map generation when multiple source files are used
to create a single output file. It could something simple as concatenating
several files, or something more complex such as joining rules with the
same selector.

### CODE REVIEW

**Comment by jonathanong** [comment]
@visionmedia https://github.com/visionmedia/css-parse/issues/35

**Comment by tj** [comment]
dope!

**Comment by tj** [comment]
on second thought it might be nicer to do this at a more root level node, but for now it should be ok

File: test/cases/keyframes.vendor.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
                   "end": {
                     "line": 2,
                     "column": 21
-                  }
+                  },
+                  "source": "keyframes.vendor.json"
```
Comment (jonathanong) [inline test/cases/keyframes.vendor.json:29]: hahaha copy pasta gone wrong?

Comment (lydell) [inline test/cases/keyframes.vendor.json:29]: Oops. Does that break the test?

Comment (jonathanong) [inline test/cases/keyframes.vendor.json:29]: yeah, no worries i fixed it

**Pull request merged**